pandas/core/generic.py

```diff
@@ -2533,7 +2533,7 @@ def to_msgpack(self, path_or_buf=None, encoding="utf-8", **kwargs):
 
         Parameters
         ----------
-        path : string File path, buffer-like, or None
+        path : str (file path), buffer-like, or None
```

Comment:
```suggestion
        path : str, filepath, buffer-like, or None
```

Reply:
Since neither 'file path' nor 'filepath' are listed as types [here](https://pandas.pydata.org/pandas-docs/stable/development/contributing_docstring.html#about-docstrings-and-standards), I've removed this and added a little clarification below.